Misc/NEWS.d/next/Build/2019-10-13-16-18-24.bpo-38468.buCO84.rst

```diff
@@ -0,0 +1,2 @@
+Use `getvar` for all still existing `sysconfig.get_config_var()` calls.

```

Comment:
Usually, changes which are only refactoring are not documented in NEWS entry, since they have no benefit for users. If you want to keep it, you must at least mention which file is modified (Python implementation of python3.9-config).